spec.bs

```diff
@@ -8375,69 +8378,66 @@ To <dfn>build trusted key value bidding signals request body</dfn> given a [=set
   1. If |compressionIdMap| does not [=map/contain=] |joiningOrigin|:
     1. [=map/Set=] |compressionIdMap|[|joiningOrigin|] to |nextCompressionGroupId|.
     1. Increment |nextCompressionGroupId| by 1.
   1. Let |compressionGroupId| be |compressionIdMap|[|joiningOrigin|].
   1. If |compressionGroupMap| does not [=map/contain=] |compressionGroupId|:
-    1. Let |compressionGroupMap|[|compressionGroupId|] be an empty [=map=], whose [=map/keys=] are integers
+    1.[=map/Set=] |compressionGroupMap|[|compressionGroupId|] to an empty [=map=], whose [=map/keys=] are integers
       and [=map/values=] are [=maps=].
-  1. Set |executionMode| to |group|'s [=interest group/execution mode=].
-  1. If |executionMode| equal to "`group-by-origin`", set |partitionId| to 0.
+  1. If |group|'s [=interest group/execution mode=] is "`group-by-origin`", set |partitionId| to 0.
   1. Otherwise:
     1. If |compressionGroupMap|[|compressionGroupId|] [=map/contains=] 0, set |partitionId| to
       [=map/size=] of |compressionGroupMap|[|compressionGroupId|].
-    1. Otherwise, set |partitionId| to sum [=list/size=] of |compressionGroupMap|[|joiningOrigin|]
-      and 1.
-  1. Set |interestGroupIdMap|[|group|'s [=interest group/name=]] to [=tuple=] of |compressionGroupId|
-    and |partitionId|.
-  1. If |compressionGroupMap|[|compressionGroupId|] does not [=map/contain=] |partitionId|:
-    1. Let |partition| be an empty [=partition=].
-    1. Set |partition|'s [=partition/id=] to |partitionId|.
-    1. Let |namespace| be an empty [=map=], whose [=map/keys=] are [=strings=] and [=map/values=] are [=list=]
-      of [=strings=].
-    1. Set |namespace|["interest_group_names"] to [|group|'s [=interest group/name=]].
-    1. Set |namespace|["bidding_keys"] to |group|'s [=interest group/trusted bidding signals keys=].
-    1. Set |partition|'s [=partition/namespace=] to |namespace|.
-    1. Let |metadata| be an empty [=map=], whose [=map/keys=] and [=map/values=] are [=strings=].
-    1. Set |metadata|["experiment_group_id"] to |experimentGroupId|.
-    1. Set |metadata|[|slotSizeParams|[0]] to |slotSizeParams|[1].
-    1. Set |partition|'s [=partition/metadata=] to |metadata|.
-    1. Set |compressionGroupMap|[|compressionGroupId|][|partitionId|] to |partition|.
+    1. Otherwise, set |partitionId| to [=list/size=] of |compressionGroupMap|[|joiningOrigin|] plus 1.
+  1. [=map/Set=] |interestGroupIdMap|[|group|'s [=interest group/name=]] to [=tuple=] of
+    (|compressionGroupId|, |partitionId|).
+  1. If |compressionGroupMap|[|compressionGroupId|] does not [=map/contain=] |partitionId|, set
+    |compressionGroupMap|[|compressionGroupId|][|partitionId|] to a [=bidding partition=] with the following items:
+    : [=bidding partition/id=]
+    :: |partitionId|
+
+    : [=bidding partition/namespace=]
+    :: "interest_group_names" → [|group|'s [=interest group/name=]]
+    :: "bidding_keys" → |group|'s [=interest group/trusted bidding signals keys=]

```

Comment:
See [this](https://infra.spec.whatwg.org/#lists) for how to define list literals, and [this](https://infra.spec.whatwg.org/#maps) for how to define map literals.
It seems this map's value is not trivial, so probably more readable to define a variable for it above for it, but not feeling strong about that.
```suggestion
    :: The [=ordered map=] «[ "interest_group_names" → « |group|'s [=interest group/name=] »,
      "bidding_keys" → |group|'s [=interest group/trusted bidding signals keys=] ]»
```

Reply:
Gotcha